### PR TITLE
React Basics: Hello World Challenge

### PR DESCRIPTION
**Pull request by PReichetanz**
rendered: https://github.com/neuefische/web-exercises/tree/react-basics-hello-world-challenge/sessions/react-basics/hello-world

## CSB

- [CSB](https://codesandbox.io/s/github/neuefische/web-exercises/tree/react-basics-hello-world-challenge/sessions/react-basics/hello-world?file=/README.md)

refers to neuefische/web-curriculum-new-format#190

### CODE REVIEW

**Comment by djfarly** [comment]
ready for review :)

File: sessions/react-basics/hello-world/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+Check the browser tab of the codesandbox: the content of the `<div>` ("Say hello...") should be visible there as well.
+
+Now, let us cherish a tradition and render "Hello world!".
+
+## Task
+
+### Render by Modifying the `App` component
+
+Render a heading with the text "Hello world!" to the browser. To do so, change the `div` into an `h1` and adapt its text accordingly.
```
Comment (CJAkkuec) [inline sessions/react-basics/hello-world/README.md:15]:
```suggestion
Render a heading with the text "Hello world!" to the browser. To do so, change the `div` into an `h1` and change its text accordingly.
```

File: sessions/react-basics/hello-world/README.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Let's go a step further and outsource the heading to its own component.
+
+To do so, create a component (i.e. function) called `HelloWorld`.
+
+Use the following hints as guideline:
+
+- In the [App.js](src/App.js), create a new function called `HelloWorld()` (it does not matter whether you write it above or below the `App` function - but not inside it!).
+- `HelloWorld()` should return the same HTML heading as in the first task above.
+- In the return statement of the `App` function, replace the `h1` with your `HelloWorld` component.
+- Look in the browser and marvel: nothing has changed, but you've just built and used your first React component!
```
Comment (CJAkkuec) [inline sessions/react-basics/hello-world/README.md:30]:
```suggestion
- Look at the browser and be amazed: nothing has changed, but you've just built and used your first React component!
```

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
LGTM 🚀 
[Here's my solution.](https://codesandbox.io/s/wizardly-wiles-6opti4?file=/src/App.js)

Just a couple of very minor notes. :)

**Pull request merged**